### PR TITLE
updates application structure

### PR DESCRIPTION
**Pull request by chrabyrd**

### CODE REVIEW

File: README.txt
Comment (robgaston) [inline README.txt:1]: @chrabyrd we should replace this with content that outlines project setup, as per https://github.com/archesproject/arches_rdm_example_project/blob/main/README.md

File: yarn.lock
Comment (robgaston) [inline yarn.lock:1]: @chrabyrd as we are moving to npm, shouldn't this be replaced with `package-lock.json`?

**Review by robgaston — CHANGES_REQUESTED** [review]
@chrabyrd this is looking very close, but we should look at updating the `README.md` and moving from yarn to npm.  thanks!

**Comment by chrabyrd** [comment]
> @chrabyrd this is looking very close, but we should look at updating the `README.md` and moving from yarn to npm. thanks!

Ah yes -- this was done before the cutover from yarn to npm. Will update! 👍

**Review by robgaston — APPROVED** [review]

**Pull request merged**